source/utilities.tex

```diff
@@ -18337,32 +18337,32 @@
 forms the logical conjunction of its template type arguments.
 
 \pnum
-For a specialization \tcode{conjunction<B1, ..., BN>},
-if there is a template type argument \tcode{Bi} for which \tcode{bool(Bi::value)} is \tcode{false},
-then instantiating \tcode{conjunction<B1, ..., BN>::value}
-does not require the instantiation of \tcode{Bj::value} for \tcode{j > i}.
+For a specialization $\tcode{conjunction<B}_1\tcode{,} \cdots\tcode{, B}_N\tcode{>}$,
```

Comment:
Yikes, this nesting is a bit horrific. How about:

    \tcode{conjuntion<$\tcode{B}_{1}$, $\dotsc$, $\tcode{B}_{N}$>}

(I suppose code commas are OK here.)

Reply:
Yes, it's horrific. That looks like an improvement, I'll try it.

Reply:
I think baseline dots are OK here (ldots or dotsc). I don't think there's any confusion with pack expansion here. I don't feel strongly, but I don't think that we need to go out of our (typographic) way here to prevent a misunderstanding.